### PR TITLE
emby4: initial addon

### PR DESCRIPTION
**Pull request by awiouy**
With Emby Server version 4 transcoding and Live TV require priced Emby Premiere license.
Therefore, Emby Server 4 is provided as a separate package/addon: emby4.

If both emby and emby4 are installed, then system.d will only start emby4.

### CODE REVIEW

File: packages/addons/service/emby4/source/system.d/service.emby4.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Emby - streaming server
+Documentation=http://emby.media
+After=network.target
+Requires=network.target
```
[the remark this answers is not in the document]
Comment (CvH) [reply packages/addons/service/emby4/source/system.d/service.emby4.service]: this prevents the start if no network is available

[the remark this answers is not in the document]
Comment (awiouy) [reply packages/addons/service/emby4/source/system.d/service.emby4.service]: Emby is useless without network, at least localhost

**Comment by awiouy** [comment]
This ensures that essential resources (sockets, mounts, localhost, etc) are available:
```
...
[Unit]
Requires=multi-user.target
After=multi-user.target
```

This starts the service at the same time as Kodi:
```
[Install]
WantedBy=kodi.target
```

**Comment by awiouy** [comment]
@CvH The reusable code for disabling a conflicting addon: 
```
def jsonrpc(request):
    return json.loads(xbmc.executeJSONRPC(json.dumps(request)))


def disable_conflicting(conficting,
        message='{that} conflicts with {this} and has been disabled'):
    is_enabled = {'jsonrpc': '2.0', 'method': 'Addons.GetAddonDetails', 'id': 1,
                  'params': {'addonid': conficting, 'properties': ['enabled']}}
    disable = {'jsonrpc': '2.0', 'method': 'Addons.SetAddonEnabled', 'id': 1,
               'params': {'addonid': conficting, 'enabled': False}}
    try:
        if jsonrpc(is_enabled)['result']['addon']['enabled']:
            this = xbmcaddon.Addon().getAddonInfo('name')
            that = xbmcaddon.Addon(conficting).getAddonInfo('name')
            jsonrpc(disable)
            dialog = xbmcgui.Dialog()
            dialog.ok(this, message.format(
                this=this, that=that))
            del dialog
    except KeyError:
        pass
```

**Comment by awiouy** [comment]
Ready to merge

**Comment by CvH** [comment]
@awiouy could you update to 4.0.0.2 ? thats the "final" version

**Comment by awiouy** [comment]
@CvH done

**Comment by awiouy** [comment]
@anoma Emby dev Luke is aware of the addon.

**Comment by anoma** [comment]
That is excellent. What license is it released under.

**Pull request merged**